src-electron/util/types.js

```diff
@@ -78,6 +78,77 @@ async function typeSizeAttribute(db, zclPackageId, at, defaultValue = null) {
   }
 }
 
+/**
+ *
+ * @param {*} value
+ * @param {*} size
+ * @returns The big endian value for a given float value padded with
+ * the given size.
+ */
+function convertFloatToBigEndian(value, size) {
+  let res = ''
+  const arrayBuffer = new ArrayBuffer(size)
+  const dataView = new DataView(arrayBuffer)
+  if (size == 4) {
+    dataView.setFloat32(0, value, false)
+  } else if (size == 8) {
+    dataView.setFloat64(0, value, false)
+  } else {
+    // Throwing a warning for unknown float type and continuing with 64bit float
+    env.logWarning('Unknown Float Type value: ' + value)
+    dataView.setFloat64(0, value, false)
+  }
+  for (let i = 0; i < size; i++) {
+    let t = dataView.getUint8(i).toString(16)
+    if (t.length == 1) {
+      t = '0' + t
+    }
+    res += t
+  }
+  return '0x' + res
+}
+
+/**
+ *
+ * @param {*} value
+ * @param {*} size
+ * @returns The big endian value for a given integer value padded with
+ * the given size.
+ */
+function convertIntToBigEndian(value, size) {
+  const arrayBuffer = new ArrayBuffer(size)
+  const dataView = new DataView(arrayBuffer)
+  if (size == 1) {
+    dataView.setInt8(0, value, false)
+  } else if (size == 2) {
+    dataView.setInt16(0, value, false)
+  } else if (size == 3) {
+    dataView.setInt32(0, value, false)
```

Comment:
This is not going to do the right thing.  If you're setting as big-endian, this will leave the first byte 0 and set the 2nd through 4th bytes to the value, right?  But then you're going to look at the 1st to 3rd bytes below.

Same thing for sizes 5, 6, 7.

Either we need to do this little-endian, or we need to adjust where we start looking for the value or something.